html/createUser.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Create User</title>
+</head>
+<body>
+    <ul>
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="userProfile.html">User Profile</a></li>
+        <li><a href="userList.html">User List</a></li>
+        <li><a href="createUser.html">Create User</a></li>
+        <li><a href="editUser.html">Edit User</a></li>
+    </ul>
+
+    <h2>Create User</h2>
```

Comment:
For the sake of accessibility, HTML pages should usually have one top-level header `h1` tag, which is just the title of the page

Reply:
(You might also put that inside of a `<header>` tag)